docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: '3'
+
+services:
+
+  scratch-map:
+    container_name: scratch_map
+    build: ${PROJECTS_HOME}/scratch-map
+    ports:
+      - "8080:8080"
+#    labels:
+#      com.centurylinklabs.watchtower.enable: "true"
+    volumes:
+      - ${PROJECTS_HOME}/scratch-map
+    networks:
+      - scratch-net
+
+
+#  watchtower:
```

Comment:
Maybe we should remove the watchtower stuff and chuck it in another branch

Reply:
Yeah, that's fine, I just remember it being handy... Will remove

Reply:
I can remove this for @dcdarrell9 since he mentioned his computer needs to reinstall everything?